src/plugins/discover/public/application/apps/main/services/use_saved_search.ts

```diff
@@ -129,6 +129,7 @@ export const useSavedSearch = ({
    */
   const refs = useRef<{
     abortController?: AbortController;
+    autoRefreshDoneCb?: AutoRefreshDoneFn;
```

Comment:
nit can this be called just `autoRefreshDone`? I think `cb` is more confusing than valuable.

Reply:
true! thx, will do!

Reply:
done that